zipline/data/minute_bars.py

```diff
@@ -441,6 +441,13 @@ def pad(self, sid, date):
         assert new_last_date == date, "new_last_date={0} != date={1}".format(
             new_last_date, date)
 
+    def set_sid_attrs(self, sid, **kwargs):
```

Comment:
If this is only being used for the sid adjustment factor, should we call this `set_sid_adjustment_factor`?